releases/kubecost.yaml

```diff
@@ -0,0 +1,92 @@
+repositories:
+  # kubecost has their own helm repo
+  - name: "kubecost"
+    url: "https://kubecost.github.io/cost-analyzer/"
+# Kubernetes incubator repo of helm charts
+  - name: "kubernetes-incubator"
+    url: "https://kubernetes-charts-incubator.storage.googleapis.com"
+
+releases:
+  #######################################################################################
+  ## kubecost                                                                          ##
+  ## Kubecost is a cost analyzer for kubernetes cloud resrouces                        ##
+  #######################################################################################
+
+  #
+  # References:
+  #   - https://github.com/kubecost/cost-analyzer-helm-chart/blob/master/cost-analyzer/values.yaml
+  #   - http://docs.kubecost.com
+  #
+  - name: "kubecost"
+    namespace: "monitoring"
+    labels:
+      chart: "kubecost"
+      repo: "github"
+      component: "kubecost"
+      namespace: "monitoring"
+      vendor: "kubecost"
+      default: "false"
+    chart: "kubecost/cost-analyzer"
+    version: "v1.26.0"
+    wait: true
+    installed: {{ env "KUBECOST_INSTALLED" | default "true" }}
+    values:
+      {{- if eq (env "PROMETHEUS_OPERATOR_INSTALLED" | default "true") "true" }}
+      - global:
+          prometheus:
+            enabled: false # If false, Prometheus will not be installed
+            fqdn: http://prometheus-operator-prometheus.monitoring.svc.cluster.local:9090
+          grafana:
+            enabled: false # If false, Grafana will not be installed
+            domainName: {{ env "PROMETHEUS_GRAFANA_ROOT_DOMAIN" | default "prometheus-operator-grafana.monitoring.svc.cluster.local" }}
+            scheme: {{ env "PROMETHEUS_GRAFANA_ROOT_SCHEME" | default "http" }}
+            proxy: false
+        grafana:
+          sidecar:
+            dashboards:
+              enabled: true
+            datasources:
+              enabled: false
+      {{- else }}
+      - global:
+          prometheus:
+            enabled: true
+          grafana:
+            enabled: true # If false, Grafana will not be installed
+        grafana:
+          sidecar:
+            dashboards:
+              enabled: true
+            datasources:
+              enabled: true
+      {{- end }}
+{{- if eq (env "PROMETHEUS_OPERATOR_INSTALLED" | default "true") "true" }}
+# References:
+#   - https://github.com/helm/charts/tree/master/incubator/raw
+#
+  - name: 'cost-analyzer-model'
+    chart: "kubernetes-incubator/raw"
+    namespace: monitoring
+    version: "0.2.3"
+    wait: true
+    force: true
```

Comment:
The entire release is bracketed by `if PROMETHEUS_OPERATOR_INSTALLED` so "installed" seemed unnecessarily redundant.

Reply:
I have confirmed that with `helmfile` version v0.54.0, setting `installed: false` does not cause the release to be deleted on `helmfile sync`. I will add it anyway since you asked for it, but be forewarned.

Reply:
> setting `installed: false` does not cause the release to be deleted on `helmfile sync`.

It doesn't work with `sync`, it works with `apply` (e.g. like `terraform apply`)

Reply:
`apply` is what should be used in CI/CD contexts